torch/csrc/cuda/Event.cpp

```diff
@@ -238,6 +238,8 @@ PyTypeObject THCPEventType = {
 };
 
 void THCPEvent_init(PyObject* module) {
+  Py_INCREF(THPEventClass);
```

Comment:
Please assert that it is non-null here. In case someone changes the init order and this has not been set yet.

Reply:
Good idea!
Updated.